Packs/HealthCheck/ReleaseNotes/3_0_0.md

```diff
@@ -0,0 +1,132 @@
+
+#### Playbooks
+
+##### HealthCheck
+
+- %%UPDATE_RN%%
```

Comment:
@tsachzim Please add the changes you made for this and everywhere in this readme where it says "%%UPDATE_RN%%"